recipes/r-smartsnp/meta.yaml

```
@@ -0,0 +1,102 @@
+{% set version = '1.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-smartsnp
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/smartsnp_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/smartsnp/smartsnp_{{ version }}.tar.gz
+  sha256: a2c39153e425985b03427b1d1eab8a239b98272922ca49592c45c965b487ffbf
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - {{ stdlib("c") }}
+  host:
+    - r-base
+    - r-rspectra
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rfast
+    - r-bootsvd
+    - r-data.table
+    - r-foreach
+    - r-vegan
+    - r-vroom
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rspectra
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rfast
+    - r-bootsvd
+    - r-data.table
+    - r-foreach
+    - r-vegan
+    - r-vroom
+
+test:
+  commands:
+    - $R -e "library('smartsnp')"           # [not win]
+    - "\"%R%\" -e \"library('smartsnp')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=smartsnp
+  license: MIT
```

Let's add a dev URL:

```suggestion
  dev_url: https://github.com/ChristianHuber/smartsnp
  license: MIT
```